src/Fixer/DeprecatedFixerInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer;
+
+/**
+ * @author Kuba Werłos <werlos@gmail.com>
+ */
+interface DeprecatedFixerInterface extends FixerInterface
+{
+    /**
+     * Returns names of fixers to use instead.
```

Comment:
I would add _", if any"_ at the end of the sentence so that it is more explicit that the returned array can be empty.